packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -132,47 +153,148 @@ fillDate(
 
-## Example unit tests
+## Example unit tests - React Testing Library
+We recommend using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) for all your unit/integration testing needs.
 
-VSP provides helper utilities to make writing tests easier. The following example unit tests illustrate some of these helpers. You can find these unit tests in the `vets-website` repo in the [veteranInformation.unit.spec.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx) file.
+```js
+// SimpleLoginForm.js
+import React from 'react';
+const SimpleLoginForm = ({ onSubmit }) => {
+  const [error, setError] = React.useState('');
+  function handleSubmit(event) {
+    event.preventDefault();
+    const {
+      usernameInput: { value: username },
+      passwordInput: { value: password },
+    } = event.target.elements;
+    if (!username) {
+      setError('username is required');
+    } else if (!password) {
+      setError('password is required');
+    } else {
+      setError('');
+      onSubmit({ username, password });
+    }
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label htmlFor="usernameInput">Username</label>
+          <input id="usernameInput" />
+        </div>
+        <div>
+          <label htmlFor="passwordInput">Password</label>
+          <input id="passwordInput" type="password" />
+        </div>
+        <button type="submit">Submit</button>
+      </form>
+      {error ? <div role="alert">{error}</div> : null}
+    </div>
+  );
+};
+export default SimpleLoginForm;
+```
 
 ```js
+// SimpleLoginForm.unit.spec.jsx
 import React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SimpleLoginForm from '../../components/SimpleLoginForm';
+describe('my-application', () => {
+  describe('SimpleLoginForm', () => {
+    it('calls onSubmit with the username and password when submit is clicked', () => {
+        const handleSubmit = sinon.spy();
+        const { getByLabelText, getByText } = render(<SimpleLoginForm onSubmit={handleSubmit} />);
+        const user = { username: 'user123', password: 'password123' };
+
+        userEvent.type(getByLabelText(/username/i), user.username);
+        userEvent.type(getByLabelText(/password/i), user.password);
+        userEvent.click(getByText(/submit/i));
+
+        expect(handleSubmit.callCount).to.equal(1); // alternatively `expect(handleSubmit.calledOnce).to.be.true()` works as well
+        expect(handleSubmit.calledWith(user)).to.be.true(); // for more explicit testing we can use `calledWithExactly` in place of `calledWith`
+    });
+  });
+});
+```
+
+We have successfully performed a "Happy Path" test for this component. Let's break down this test:
 
```

Comment:
To break this section for better readability, we could use subheaders describing what each part is doing. Something like "Setup", "DOM interactions and queries", and "Assertions".

```suggestion

#### Setup

```

Reply:
- [x] Formatting update